test/integration/render-tests/regressions/mapbox-gl-js#4647/style.json

```diff
@@ -0,0 +1,55 @@
+{
+  "version": 8,
+  "metadata": {
+    "test": {
+      "height": 64,
+      "width": 64,
+      "operations": [
+        [
+          "wait"
+        ],
+        [
+          "showCollisionBoxes"
```

Comment:
https://github.com/mapbox/mapbox-gl-js/blob/878008895104b67861ce73f65809f7f6f0ed7263/test/integration/render-tests/debug/collision/style.json#L8-L8 is similar, but with different terminology. Can we make consistent?